orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/job/RunJobStage.groovy

```diff
@@ -34,6 +34,9 @@ class RunJobStage implements StageDefinitionBuilder {
     builder
       .withTask("runJob", RunJobTask)
       .withTask("monitorDeploy", MonitorKatoTask)
-      .withTask("waitOnJobCompletion", WaitOnJobCompletion)
+
+    if (!stage.getContext().containsKey("ignoreResults") || (Boolean.parseBoolean(stage.getContext().get("ignoreResults").toString()) == true)) {
```

Comment:
Tweaked the `code` above to be strict about only skipping if `waitForCompletion == "false"` rather than `waitForCompletion != "true"` ... seems slightly safer